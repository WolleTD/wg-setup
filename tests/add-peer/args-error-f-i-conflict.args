-f /etc/wireguard/wg0.conf -i wg0
