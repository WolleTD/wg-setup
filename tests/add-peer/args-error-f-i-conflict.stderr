Error: Can't use -i and -f together!
